doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -247,16 +247,16 @@ int main(int argc, char** argv)
 
   geometry_msgs::Pose target_pose3 = start_pose2;
 
-  target_pose3.position.z += 0.2;
-  waypoints.push_back(target_pose3);  // up
+  target_pose3.position.z -= 0.2;
+  waypoints.push_back(target_pose3);  // down
 
-  target_pose3.position.y -= 0.1;
-  waypoints.push_back(target_pose3);  // left
+  target_pose3.position.y -= 0.2;
+  waypoints.push_back(target_pose3);  // right
```

Comment:
Technically this is subjective but I'll allow it :stuck_out_tongue_closed_eyes:

Reply:
If the tutorial didn't mention the box appearing on the arm's right a little bit later in the lesson I would have left this alone :P